auth_brute_force/__openerp__.py

```diff
@@ -0,0 +1,42 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
```

Comment:
Please switch to [shortened header](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3) for all files